### ISSUE TITLE
Error during 'Vagrant up' command

### ISSUE THREAD

**Issue by somus** [mdkholy/dashbrew#4]
After cloning the repository, I ran `vagrant up`. This is the output i got

```
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'mdkholy/dashbrew'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'mdkholy/dashbrew' is up to date...
==> default: Setting the name of the VM: dashbrew_default_1425478812212_3545
==> default: Fixed port collision for 22 => 2222. Now on port 2201.
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 22 => 2201 (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2201
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection timeout. Retrying...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
==> default: Setting hostname...
==> default: Configuring and enabling network interfaces...
==> default: Mounting shared folders...
    default: /var/www => /Users/somu/dashbrew/public
    default: /vagrant => /Users/somu/dashbrew
==> default: Running provisioner: shell...
    default: Running: /var/folders/by/8r45z959363gydsb7tncnxy40000gn/T/vagrant-shell20150304-78857-11mzrv9.sh
==> default: stdin: is not a tty
==> default: /tmp/vagrant-shell: line 23: /vagrant/provision/main/dashbrew: Permission denied
==> default: Running provisioner: hostsupdate...
==> default: Updating hosts file on the guest machine...
==> default: Unable to find the file '/provision/main/etc/hosts.json'
==> default: '/etc/hosts' file updated successfully
==> default: Updating hosts file on the host machine (password may be required)...
==> default: Unable to find the file '/provision/main/etc/hosts.json'
```

I checked the provision/main/etc folder, there is no hosts.json file.

**Comment by chrisRidgers** [comment]
+1 same

**Comment by johnRivs** [comment]
Same thing here:

```
==> default: Running provisioner: hostsupdate...
==> default: Updating hosts file on the guest machine...
==> default: Unable to find the file '/provision/main/etc/hosts.json'
==> default: Updating hosts file on the host machine (password may be required)...
==> default: Unable to find the file '/provision/main/etc/hosts.json'
```

**Comment by mohatt** [comment]
What is your OS and Vagrant version?

**Comment by Brunty** [comment]
I had this issue as well, the problem being that provision/main/dashbrew couldn't be executed as it wasn't set as executable.

`$ chmod +x provision/main/dashbrew`

Seemed to do the trick for me.

**Comment by mohatt** [comment]
Yes, that should fix it :). I just want know what OS that causes this issue because I can reproduce it on my Windows machine.

**Comment by Brunty** [comment]
I had this issue on OS X (10.10)

**Comment by mohatt** [comment]
Fixed in #8. Please update your working copy and try again :).
